GettingStarted/GettingStarted.md

```diff
@@ -37,6 +38,19 @@ Once the VPN connection is established, you should be able to navigate to the th
 
 > At the end of your JupyterLab session, please make sure you save all and log out (from the launcher menu), to free up the cluster for others.
 
+## Accessing the DC2 data set using the Jupyter notebook instance running at NCSA 
+For Stack Club members using the DC2 data sets at the NCSA, there is an additional step they must do to authorize their account to access the DC2 data. The step is explained in the file /repo/README.md in the NSCA RSP (you can see this README.md in your linux terminal by typing ```cat /repo/README.md``` ).  Specifically, the user must create a small file named ```~/.pgpass``` .  That file must contain one line: 
+
+```
+lsst-pg-prod1.ncsa.illinois.edu:5432:lsstdb1:myusername:mysecretpasswd
+```
+where the ```myusername``` is your NCSA username, and ```mysecretpassword``` is your NCSA password.  
+
+
+Now if you wish, you can import the notebooks developed for DP0 into your instance of Jupyter notebook at NCSA, by issuing a git clone 
+command ```git clone https://github.com/rubin-dp0/tutorial-notebooks.git``` and now you can use the DP0 - developed notebooks on the DC2 data.  
+You might wish to create a separate directory for the DP0-developed material, but this depends how you organize your Rubin notebooks.  
+It is important to note that the location of the data repository accessible from IDF (``` s3://butler-us-central1-dp01 ```) is not the same as that accessible from NCSA (``` /repo/dc2 ```). 
```

Comment:
You also have to add an instrument argument to the butler constructor.  I think I'd prefer to have the changes users need to make be very explicit i.e. instead of 
```repo = 's3://butler-us-central1-dp01'```
you need to change to
```repo = '/repo/dc2'```
and
You must provide an instrument in and data id e.g. `'instrument': 'LSSTCam-imSim'`

Reply:
@kadrlica - in response to your query about repos, I think that the repo difference is in my i, but we might as well add the other comment from Simon, namely You must provide an instrument in and data id e.g. 'instrument': 'LSSTCam-imSim' .  Otherwise, I think it is ready for a merge.  I like your grammar fixes!